homeassistant/components/humidifier/strings.json

```diff
@@ -74,5 +74,38 @@
     "humidifier": {
       "name": "[%key:component::humidifier::entity_component::_::name%]"
     }
+  },
+  "services": {
+    "set_mode": {
+      "name": "Set mode",
+      "description": "Sets humidifier operation mode.",
+      "fields": {
+        "mode": {
+          "description": "Operation mode."
+        }
+      }
+    },
+    "set_humidity": {
+      "name": "Set humidity",
+      "description": "Sets the target humidity.",
+      "fields": {
+        "humidity": {
+          "name": "Humidity",
+          "description": "Target humidity."
```

Comment:
```suggestion
          "description": "Target humidity (in percent)."
```

Reply:
The unit is already provided by the selector

Reply:
Thank you. Rephrased.